.github/CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 /bindings/payloads/xml/ @egekorkan @mjkoster
 /bindings/platforms/hue/ @egekorkan @mjkoster
-/bindings/protocols/bacnet/ @ektrah @mjkoster
+/bindings/protocols/bacnet/ @ektrah @mjkoster @fennibay 
```

Comment:
@fennibay I think _someone_ needs no add you as contributor to this GitHub repo

Reply:
The proposal is to wait to merge this PR, at that point @fennibay will be an editor, thus making sense to give him write access to the repo.